serveradmin/servershell/static/js/servershell.js

```diff
@@ -216,7 +216,7 @@ servershell.alert = function(text, level, auto_dismiss=5) {
     box.addClass(`alert-${level}`);
 
     let message = $('#js-alert-message');
-    message.html(text);
+    message.text(text);
```

Comment:
Looking at line 195 here, the documentation says that it can have html. I wonder if it was somewhere already used with html.
My suggestion would be that it could still have html, but not by default and one need to add a parameter like `html=true` to tell the method to use `html` and not `text`.

Reply:
Only place that I can see that **could** return HTML would be:

- https://github.com/innogames/serveradmin/blob/master/serveradmin/servershell/static/js/servershell/command.js#L585

I am in favour for security here and better just allow only plain text. I will adjust the doc instead.

Reply:
If everything just uses `text` then even better. Sure text is just safer. Just saying, if html is needed, I would go for the opt-in html parameter. Adjusting the doc for now is totally good.